README.md

```diff
@@ -869,11 +869,11 @@ either the `with_params` chain method in the `run` matcher
 it { is_expected.to run.with_params('foo', 'bar', ['baz']) }
 ```
 
-Or by using the `call` method on the subject directly
+Or by using the `execute` method on the subject directly
 
 ```ruby
 it 'something' do
-  subject.call(['foo', 'bar', ['baz']])
+  subject.exectue('foo', 'bar', ['baz'])
```

Comment:
spelling

Reply:
Fixed in https://github.com/rodjek/rspec-puppet/pull/733

Reply:
Sorry :(